src/Core/src/Platform/Android/TextAlignmentExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Android.Widget;
+using AGravityFlags = Android.Views.GravityFlags;
+using ATextAlignment = Android.Views.TextAlignment;
+
+namespace Microsoft.Maui
+{
+	internal static class TextAlignmentExtensions
+	{
+		public static ATextAlignment ToTextAlignment(this TextAlignment alignment)
+		{
+			switch (alignment)
+			{
+				case TextAlignment.Center:
+					return ATextAlignment.Center;
+				case TextAlignment.End:
+					return ATextAlignment.ViewEnd;
+				default:
+					return ATextAlignment.ViewStart;
+			}
+		}
+
+		public static AGravityFlags ToVerticalGravityFlags(this TextAlignment alignment)
```

Comment:
Are you also planning on porting `ToHorizontalGravityFlags()`?

Reply:
I think the horizontal uses `ToTextAlignment`?

But, looking at the old renderer, seems we don't touch the horizontal alignment:

```
if ((int)Build.VERSION.SdkInt < 17 || !view.Context.HasRtlSupport())
{
	view.Gravity = vertical.ToVerticalGravityFlags() | horizontal.ToHorizontalGravityFlags();
}
else
{
	view.TextAlignment = horizontal.ToTextAlignment();
	view.Gravity = vertical.ToVerticalGravityFlags();
}
```

Are we safe with the RTL check gone?